repose-aggregator/components/filters/uri-stripper-filter/src/main/scala/org/openrepose/filters/uristripper/UriStripperFilter.scala

```diff
@@ -68,7 +68,7 @@ class UriStripperFilter @Inject()(configurationService: ConfigurationService)
   private var templateMapResponse: Map[LinkPath, Templates] = _
   private val DROP_CODE: String = "[[DROP]]"
 
-  implicit def toLogController(c : Controller) = new {
+  def toLogController(c : Controller) = new {
```

Comment:
If we are dropping the `implicit` here, which I am totally cool with since we only need it in this class, do we need to even both with the new extended object funkiness? May as well just call the method we want to add directly. In which case this method can just become:

```
def addLogErrorListener(c: Controller): Unit = {
  c.asInstanceOf[Transformer].setErrorListener (new LogErrorListener)
  c.setMessageEmitter(new MessageWarner())
}
```

Reply:
I really didn't want to mess with it further.